tool/tsh/kube_test.go

```diff
@@ -73,6 +74,14 @@ func TestListKube(t *testing.T) {
 			require.NoError(t, err)
 			return len(rootClusters) >= 2
 		}),
+		withNewTeleportOption(
```

Comment:
I think these kube changes were a part of Tiago's fix? I think you need to rebase your fork off of latest.